Feature: friendly error while running 'run workflow' without workflows configured

**Is your feature request related to a problem? Please describe.**
I was testing a new workflow plugin and got some weird slack error. Turns out that I forgot to configure workflows for this new workflow plugin. 

**Describe the solution you'd like**
Show a private message to the slack user trying to run a workflow that there are no workflows configured.

**Error in slack**
```
Sorry, we've run into an unexpected error. For help please reach out to your Dispatch admins and provide them with the following token: '****'
```

**Error in logs**
```
slack_1      |     raise e.SlackApiError(message=msg, response=self)
slack_1      | slack_sdk.errors.SlackApiError: The request to the Slack API failed. (url: https://www.slack.com/api/views.open)
slack_1      | The server responded with: {'ok': False, 'error': 'invalid_arguments', 'response_metadata': {'messages': ['[ERROR] failed to match all allowed schemas [json-pointer:/view]', '[ERROR] must provide at least 1 items [json-pointer:/view/blocks/1/elements/0/options]']}}
slack_1      | DEBUG:Sending a request - url: https://www.slack.com/api/chat.postEphemeral, query_params: {}, body_params: {}, files: {}, json_body: {'channel': '****', 'user': '*****', 'text': "Sorry, we've run into an unexpected error. For help please reach out to your Dispatch admins and provide them with the following token: '******'.", 'blocks': None}, headers: {'Content-Type': 'application/json;charset=utf-8', 'Authorization': '(redacted)', 'User-Agent': 'Python/3.9.7 slackclient/3.11.2 Linux/5.13.16-200.fc34.x86_64'}:/usr/local/lib/python3.9/site-packages/slack_sdk/web/base_client.py:_urllib_api_call:252

```

--- a/src/dispatch/plugins/dispatch_slack/modals/workflow/handlers.py
+++ b/src/dispatch/plugins/dispatch_slack/modals/workflow/handlers.py
@@ -34,6 +34,7 @@ def create_run_workflow_modal(
 
     incident = incident_service.get(db_session=db_session, incident_id=incident_id)
     workflows = workflow_service.get_enabled(db_session=db_session)
+    workflows = [x for x in workflows if x.plugin_instance.enabled]
 
     if workflows:
         modal_create_template = run_workflow_view(incident=incident, workflows=workflows)
@@ -47,7 +48,7 @@ def create_run_workflow_modal(
                 "type": "section",
                 "text": {
                     "type": "mrkdwn",
-                    "text": "No workflows are enabled. You can enable one in the Dispatch UI at /workflows.",
+                    "text": "No workflows are enabled or workflows plugin is not enabled. You can enable one in the Dispatch UI at /workflows.",
                 },
             }
         ]

--- a/src/dispatch/plugins/dispatch_slack/modals/workflow/views.py
+++ b/src/dispatch/plugins/dispatch_slack/modals/workflow/views.py
@@ -46,10 +46,6 @@ def run_workflow_view(
     selected_option = None
     workflow_options = []
     for w in workflows:
-        # don't show disable workflows or workflows with disabled plugins
-        if not w.plugin_instance.enabled or not w.enabled:
-            continue
-
         current_option = {
             "text": {
                 "type": "plain_text",

